data/strings/strings.txt

```diff
@@ -22172,220 +22172,177 @@
     zh-Hans = 绕行设置
     zh-Hant = 繞行設定
 
-  [driving_options_subheader]
-    tags = android,ios
-    en = Avoid on every route
-    af = Vermy op elke roete
-    ar = تجنبها على جميع الطرق
-    be = Пазбягаць на кожным маршруце
-    bg = Избягване при всеки маршрут
-    ca = Evita-ho en totes les rutes
-    cs = Vyhněte se na každé trase
-    da = Undgå på enhver rute
-    de = In jeder Reiseroute vermeiden
-    el = Αποφυγή σε κάθε διαδρομή
-    es = Evitar en todas las rutas
-    es-MX = Evitar en cada ruta
-    et = Väldi igal marsruudil
-    eu = Ibilbide guztietan saihestu
-    fa = اجتناب از تمامی مسیرها
-    fi = Vältettävä kaikissa reiteissä
-    fr = Éviter sur tous les itinéraires
-    hu = Elkerülés minden útvonalon
-    id = Hindari di setiap rute
-    it = Evitare in tutti i percorsi
-    ja = すべてのルートで回避
-    ko = 모든 길에서 피하기
-    lt = Vengti kiekviename maršrute
-    mr = प्रत्येक मार्गावर टाळा
-    nb = Unngå ved hver rute
-    nl = Vermijden op elke route
-    pl = Unikaj na każdej trasie
-    pt = Evitar em todos os percursos
-    pt-BR = Evitar em todas as rotas
-    ro = De evitat pe orice traseu
-    ru = Избегать в каждом маршруте
-    sk = Vyhnúť sa na každej trase
-    sv = Undvik på varje rutten
-    sw = Epuka kwenye kila njia
-    th = หลีกเลี่ยงในทุกเส้นทาง
-    tr = Her rotada kaçının
-    uk = Уникати в кожному маршруті
-    vi = Tránh trên mỗi tuyến
-    zh-Hans = 在每条线路上规避
-    zh-Hant = 在每條線路上規避
-
   [avoid_tolls]
     tags = android,ios
-    en = Toll roads
-    af = Tolpaaie
-    ar = الطرق ذات الرسوم
-    be = Платныя дарогі
-    bg = Платени пътища
-    ca = Vies de peatge
-    cs = Silnice s mýtným
-    da = Betalingsveje
-    de = Mautstraßen
-    el = Δρόμοι με διόδια
-    es = Carreteras de pago
-    es-MX = Carreteras de peaje
-    et = Tasulised teed
-    eu = Ordainpeko errepideak
-    fa = جاده‌های دارای عوارض
-    fi = Maksulliset tiet
-    fr = Routes à péage
-    hu = Díjköteles utak
-    id = Jalan tol
-    it = Strade a pedaggio
-    ja = 有料道路
-    ko = 유로 도로
-    lt = Mokami keliai
-    mr = टोल मार्ग
-    nb = Bompengeveier
-    nl = Tolwegen
-    pl = Drogi płatne
-    pt = Estradas com portagem
-    pt-BR = Pedágios
-    ro = Drumuri cu plată
-    ru = Платные дороги
-    sk = Spoplatnené cesty
-    sv = Betalväg
-    sw = Barabara za kulipia
-    th = ถนนแบบเสียค่าผ่านทาง
-    tr = Paralı yollar
-    uk = Платні дороги
-    vi = Đường trả phí
-    zh-Hans = 收费公路
-    zh-Hant = 收費公路
+    en = Avoid toll roads
+    af = Vermy tolpaaie
+    ar = تجنب الطرق ذات الرسوم
+    be = Пазбягайце платных дарог
+    bg = Избягвайте платените пътища
```

Comment:
The current translation for some languages is incorrect, it says "[You] avoid tolls" instead of neutral "to avoid tolls", and also uses the wrong form for the subject. I've generated some with `tools/python/translate.py` when using an appropriate neutral form as a base (non-English language), you may replace all non-proof read translations with better ones (looks like Bulgarian is still in the wrong form though):

Also note the short form in English: Avoid tolls (like in GMaps). Let's try to keep OM succinct and simple ;-)

```
    en = Avoid tolls
    ar = تجنب الطرق ذات الرسوم
    be = Пазбягаць платныя дарогі
    bg = Избягвайте платените пътища
    ca = Eviteu les autopistes de peatge
    cs = Vyhněte se zpoplatněným silnicím
    da = Undgå betalingsveje
    de = Mautstraßen meiden
    el = Αποφύγετε τους δρόμους με διόδια
    es = Evita las autopistas de peaje
    es-MX = Evite las carreteras de peaje
    et = Vältige tasulisi teid
    eu = Saihestu ordainpeko errepideak
    fa = از جاده های عوارضی خودداری کنید
    fi = Vältä maksullisia teitä
    fr = Évite les routes à péage
    he = הימנעו מכבישי אגרה
    hi = टोल सड़कों से बचें
    hu = Kerülje a fizetős utakat
    id = Hindari jalan tol
    it = Evita le strade a pedaggio
    ja = 有料道路を避ける
    ko = 유료 도로 피하기
    mr = टोल रस्ते टाळा
    nb = Unngå bompenger
    nl = Vermijd tolwegen
    pl = Unikaj płatnych dróg
    pt = Evita as estradas com portagem
    pt-BR = Evite estradas com pedágio
    ro = Evitați drumurile cu taxă
    ru = Избегать платные дороги
    sk = Vyhnite sa spoplatneným cestám
    sv = Undvik avgiftsbelagda vägar
    sw = Epuka barabara za ushuru
    th = หลีกเลี่ยงถนนที่เก็บค่าผ่านทาง
    tr = Ücretli yollardan kaçının
    uk = Уникати платних доріг
    vi = Tránh đường thu phí
    zh-Hans = 避免使用收费公路
    zh-Hant = 避開收費公路
```